docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -20,401 +51,388 @@ The scenarios of defining placement rules in SQL include:
 
 ```sql
 [server]
-labels = "zone=bj,rack=rack0,disk=hdd"
+labels = "region=us-east-1,rack=rack0,disk=hdd"
 ```
 
-Then `+zone=bj` matches this store while `+disk=ssd` doesn't.
+Then `+region=us-east-1` matches this store while `+disk=ssd` doesn't.
 
-In the dictionary format, `count` must be specified, which indicates the number of replicas placed on those stores. When the prefix is `-`, the `count` is still meaningful.
+In the dictionary format, `count` must be specified, which indicates a quantity which must match. When the prefix is `-`, the `count` is still meaningful.
 
-For example, `CONSTRAINTS="{+zone=sh:1,-zone=bj:2}"` indicates to place 1 replica in `sh`, 2 replicas in anywhere but `bj`.
+For example, `FOLLOWER_CONSTRAINTS="{+region=us-east-1:1,-region=us-east-2:2}"` indicates to place at least 1 follower in `us-east-1`, 2 replicas in anywhere but `us-east-2` (by definition this will be `us-east-1` since there are no other regions available).
 
-In the list format, `count` is not specified. The number of replicas for each constraint is not limited, but the total number of replicas should still conform to the `REPLICAS` option.
+In the list format, `count` is not specified. The number of followers for each constraint is not limited, but the total number of instances should still conform to the definition.
 
-For example, `CONSTRAINTS="[+zone=sh,+zone=bj]" REPLICAS=3` indicates to place 3 replicas on either `sh` or `bj`. There may be 2 replicas on `sh` and 1 in `bj`, or 2 in `bj` and 1 in `sh`. It's up to PD.
+For example, `FOLLOWER_CONSTRAINTS="[+region=us-east-1,+region=us-east-2]" FOLLOWERS=3` indicates to place 3 followers on either `us-east-1` or `us-east-1`. There may be 2 replicas on `us-east-1` and 1 in `us-east-2`, or 2 in `us-east-2` and 1 in `us-east-1`. It's up to PD (see "Schedule Property" for additional details).
 
 Label constraints can be implemented by defining `label_constraints` field in PD placement rule configuration. `+` and `-` correspond to property `op`. Specifically, `+` is equivalent to `in` and `-` is equivalent to `notIn`.
 
-For example, `+zone=sh,+zone=bj,-disk=hdd` is equivalent to:
+For example, `+region=us-east-1,+region=us-east-2,-disk=hdd` is equivalent to:
 
 ```
 "label_constraints": [
-	{"key": "zone", "op": "in", "values": ["sh", "bj"]},
+	{"key": "region", "op": "in", "values": ["us-east-1", "us-east-2"]},
 	{"key": "disk", "op": "notIn", "values": ["hdd"]}
 ]
 ```
 
 Field `location_labels` in PD placement rule configuration is used to isolate replicas to different zones to improve availability. For now, the global configuration can be used as the default `location_labels` for all placement rules defined in SQL, so it's unnecessary to specify it.
 
-### Role configuration
+`PLACEMENT` also supports adding TiFlash replicas for a table, as the statement `ALTER TABLE table_name SET TIFLASH REPLICA count` does. For example:
```

Comment:
What is the expected relation between placement rule in SQL and the 'SET REPLICA' statement?

```
ALTER TABLE table_name SET TIFLASH REPLICA count
```

Reply:
The current impl ignore tiflash, because it is not verified that placement rules works with tiflash.

Reply:
Is there any known issue/concern for Placement Rules in SQL and TiFlash?

Reply:
Replicas does not work well for tiflash, since the standard `set tiflash replicas` requires another HTTP api access. And schedule did not work for tiflash too due to key format, but it is fixed in the very early stage.